package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@artsy/passport",
-  "version": "1.1.11",
-  "description": "Wires up the common auth handlers for Artsy's [Ezel](ezeljs.com)-based apps using [passport](http://passportjs.org/).",
+  "version": "1.1.15",
```

Comment:
Why not 1.1.12?

Reply:
Good catch that should be 1.1.12, I'll update 👍